prov/rxm/src/rxm_cq.c

```diff
@@ -1721,9 +1721,14 @@ void rxm_handle_comp_error(struct rxm_ep *rxm_ep)
 		return;
 	}
 
-	if (err_entry.err != FI_ECANCELED)
-		OFI_CQ_STRERROR(&rxm_prov, FI_LOG_WARN, FI_LOG_CQ,
-				rxm_ep->msg_cq, &err_entry);
+	if (err_entry.err != FI_ECANCELED) {
+		FI_WARN(&rxm_prov, FI_LOG_CQ,
```

Comment:
FI_WARN is one statement, no brace around it.

Reply:
Same as above.